lib/std/os/windows/win32error.zig

```diff
@@ -1,5 +1,27 @@
 // Codes are from https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-erref/18d8fbe8-a967-4f1c-ae50-99ca8e491d2d
 pub const Win32Error = extern enum(u16) {
+    const fmt = @import("std").fmt;
+
+    pub fn format(
+        self: Win32Error,
+        comptime fmt_str: []const u8,
+        options: fmt.FormatOptions,
+        writer: var,
+    ) !void {
+        try writer.writeAll(@typeName(Win32Error));
```

Comment:
I'm not familair with `@setEvalbranchQuota`, but would it be possible to set that and then call back into the common enum formatting code?

Reply:
Or another idea, would it be possible to set this quote in the common enum formatting code, based on the enum size?

Reply:
> but would it be possible to set that and then call back into the common enum formatting code?

No, as the code used to format enums normally checks if the type has a "format" function and uses that - would cause a dependency loop.

> would it be possible to set this quota in the common enum formatting code, based on the enum size?

Yes this could work as an alternative to this PR, but I was hesitant to do so

Reply:
This seems to work:
```diff
 
             // Use @tagName only if value is one of known fields
+            @setEvalBranchQuota(2 * enumInfo.fields.len + 6);
             inline for (enumInfo.fields) |enumField| {
                 if (@enumToInt(value) == enumField.value) {
```

However, I'm not sure why.  I don't know what a "backward branch" is.  If I set it to `+ 5` then it doesn't work.  Also if I add more values to the enum, it also doesn't change the limit of what works and what doesn't.

Reply:
> No, as the code used to format enums normally checks if the type has a "format" function and uses that - would cause a dependency loop.

Could be solved by putting the code into a function, but if we can find a way to use `@setEvalBranchQuota` that would likely be better since it would support all enums without having to patch the format code to make it work for large enums.